Frameworks/include/CALayerInternal.h

```diff
@@ -115,20 +116,16 @@ class CAPrivateInfo : public CADisplayProperties, public LLTreeNode<CAPrivateInf
 // Kicks off an update to the layer's layout and display hierarchy if needed
 - (void)_displayChanged;
 
-- (void)updateAccessibilityInfo:(const IWAccessibilityInfo*)info;
++ (CGPoint)convertPoint:(CGPoint)point fromLayer:(CALayer*)layer toLayer:(CALayer*)layer;
 
 - (void)_removeAnimation:(CAAnimation*)animation;
-- (DisplayTexture*)_getDisplayTexture;
+- (std::shared_ptr<IDisplayTexture>)_getDisplayTexture;
 
 - (CAPrivateInfo*)_priv;
 - (void)_setRootLayer:(BOOL)isRootLayer;
 
 - (void)_setZIndex:(int)zIndex;
```

Comment:
Another property only placed here because it is used by UIWindow - file a bug so we can clean it up and integrate it more cleanly. We removed all other z-methods (zPosition, anchorPointZ, etc.) from CALayer